packages/content-conformance/src/__tests__/cli.test.ts

```diff
@@ -0,0 +1,222 @@
+import path from 'path'
+import { execFileSync } from 'child_process'
+import { currentFilePath } from '../test/utils'
+
+const baseFixturePath = path.join(
+  currentFilePath,
+  '../..',
+  './__tests__/__fixtures__/basic-with-content-files'
+)
```

Comment:
[suggestion] instead of exporting and using `currentFilePath`, there's a utility from that file that can provide a path to a fixture:

```ts
import { getFixturePath } from '../test/utils'

const baseFixturePath = getFixturePath('basic-with-content-files')
```